Update job-scheduler.md (#4676)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes/79-July-2023/faster-run.md

Our enhanced scheduler offers more durability and empowers users to run jobs effortlessly. 

This means Enterprise, multi-tenant accounts can now enjoy the advantages of unlimited job concurrency. Previously limited to a fixed number of run slots, Enterprise accounts now have the freedom to operate without constraints. Single-tenant support will be coming soon.

Something to note, each running job occupies a run slot for its duration, and if all slots are occupied, jobs will queue accordingly.

For more feature details, refer to the [dbt Cloud pricing page](https://www.getdbt.com/pricing/).

Note, Team accounts created after July 2023 benefit from unlimited job concurrency:
- Legacy Team accounts have a fixed number of run slots.

> - Project limitations &mdash; Both Team and Developer plans are limited to one project each. For larger-scale needs, our [Enterprise plan](https://www.getdbt.com/pricing/) offers features such as audit logging, unlimited job concurrency and projects, and more.

Reviewer: ```suggestion
- Both Team and Developer plans are limited to one project each. For larger-scale needs, our [Enterprise plan](https://www.getdbt.com/pricing/) offers features such as audit logging, unlimited job concurrency and projects, and more.
```

This means Enterprise, multi-tenant accounts can now enjoy the advantages of unlimited job concurrency. Previously limited to a fixed number of run slots, Enterprise accounts now have the freedom to operate without constraints. Single-tenant support will be coming soon. Team plan customers will continue to have only 2 run slots. 

Something to note, each running job occupies a run slot for its duration, and if all slots are occupied, jobs will queue accordingly.

For more feature details, refer to the [dbt Cloud pricing page](https://www.getdbt.com/pricing/).

> Note, newly created Team accounts after xyz can now benefit from unlimited job concurrency:

Reviewer: ```suggestion
Note, newly created Team accounts after July 2023 benefit from unlimited job concurrency:
```

- Existing Team plans &mdash; Existing Team plan accounts will continue to operate with their original fixed number of run slots.

> - Project limitations &mdash; Both Team and Developer plans are limited to one project each. For larger-scale needs, our [Enterprise plan](https://www.getdbt.com/pricing/) provides unlimited job concurrency and project capacity. 

Reviewer: ```suggestion
- Project limitations &mdash; Both Team and Developer plans are limited to one project each. For larger-scale needs, our [Enterprise plan](https://www.getdbt.com/pricing/) offers features such as audit logging, unlimited job concurrency and projects, and more.
```

This means Enterprise, multi-tenant accounts can now enjoy the advantages of unlimited job concurrency. Previously limited to a fixed number of run slots, Enterprise accounts now have the freedom to operate without constraints. Single-tenant support will be coming soon. Team plan customers will continue to have only 2 run slots. 

Something to note, each running job occupies a run slot for its duration, and if all slots are occupied, jobs will queue accordingly.

For more feature details, refer to the [dbt Cloud pricing page](https://www.getdbt.com/pricing/).

- **Update: New Team plans with unlimited job concurrency**<br />

> - Unlimited Job concurrency for new Team plans &mdash; New accounts on the Team plan now benefit from unlimited job concurrency.

Reviewer: ```suggestion
```

This means Enterprise, multi-tenant accounts can now enjoy the advantages of unlimited job concurrency. Previously limited to a fixed number of run slots, Enterprise accounts now have the freedom to operate without constraints. Single-tenant support will be coming soon. Team plan customers will continue to have only 2 run slots. 

Something to note, each running job occupies a run slot for its duration, and if all slots are occupied, jobs will queue accordingly.

For more feature details, refer to the [dbt Cloud pricing page](https://www.getdbt.com/pricing/).

> - **Update: New Team plans with unlimited job concurrency**<br />

Reviewer: ```suggestion
Note, newly created Team accounts after xyz can benefit can now benefit from unlimited job concurrency:
```

> - **Update December 2023: New Team plans with unlimited job concurrency**<br />

Reviewer: ```suggestion
- **Update: New Team plans with unlimited job concurrency**<br />
```

>   We've introduced a change to our dbt Cloud Scheduler for newly created Team plan accounts:<br /><br />

Reviewer: ```suggestion
```
